tracks/javascript/exercises/leap/README.md

```diff
@@ -0,0 +1,33 @@
+### Reasonable solutions
+
+```javascript
+module.exports = {
+  isLeap(year) {
+        if (this.year % 4 !== 0) return false
+
+        if (this.year % 100 !== 0) return true
+
+        if (this.year % 100 === 0 && this.year % 400 === 0) return true
+
+        return false
+  }
```

Comment:
I think a talking point would be that if statements that return true and false probably shouldn't because conditional statements are by definition true or false.